ingress-gateway-ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I believe this file is not required, sorry I need to request changes until this file is removed